sdgym/benchmark.py

```diff
@@ -325,15 +353,27 @@ def _score_with_timeout(
         return output
 
 
-def _format_output(output, name, dataset_name, compute_quality_score, cache_dir):
-    evaluate_time = None
-    if 'scores' in output or 'quality_score_time' in output:
-        evaluate_time = output.get('quality_score_time', 0)
+def _format_output(
+    output, name, dataset_name, compute_quality_score, compute_diagnostic_score, cache_dir
+):
+    evaluate_time = 0
+    print(output)
```

Comment:
Remove? 
```suggestion
```

Reply:
I see our lint for this repo doesn't catch print statements lol

Reply:
Fix is here:
https://github.com/sdv-dev/SDGym/pull/318